localstripe/resources.py

```diff
@@ -935,7 +935,10 @@ def __init__(self, customer=None, subscription=None, metadata=None,
         self._draft = True
         self._voided = False
 
-        if not simulation:
+        if not simulation and not upcoming:
+            if subscription is not None:
+                subscription_obj.latest_invoice = self.id
+
             schedule_webhook(Event('invoice.created', self))
```

Comment:
A small question: the condition to execute this line seems to have changed (condition `not upcoming` was added).
Thus it changes when this webhook is send?

Reply:
Yes, totally right!

Before, an `invoice.created` event was sent the user looked at the next upcoming invoice (`GET /v1/invoices/upcoming?customer=cus_n2FydPZCLKMDXk`).
Now, it's not the case anymore (and I think it makes more sense, but feel free to comment if I'm wrong).

Reply:
Indeed it make sense, thanks for the answer!